src/scene/graphics/render-pass-depth.js

```diff
@@ -0,0 +1,11 @@
+import { RenderPass } from "../../platform/graphics/render-pass.js";
```

Comment:
Is this meant to be in the PR?

Reply:
nope :(
too keen to get that rename going. This is next PR, sorry.